src/main/java/uk/gov/hmcts/reform/ccd/client/CaseEventsApi.java

```diff
@@ -14,7 +14,7 @@
 import static uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi.SERVICE_AUTHORIZATION;
 
 @FeignClient(
-    name = "core-case-data-api",
+    name = "core-case-event-api",
```

Comment:
do you also need to remove https://github.com/hmcts/ccd-client/blob/master/src/test/resources/application.yaml